src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -388,3 +388,12 @@
         az keyvault update --enabled-for-disk-encryption true --name MyKeyVault --resource-group MyResourceGroup
     crafted: true
 """
+
+helps['keyvault wait'] = """
```

Comment:
is this a new command? or just fix of help file?

Reply:
> is this a new command? or just fix of help file?

@yungezz This is for LRO. `keyvault create` and `keyvault purge` became LRO in track 2.